misc/python/materialize/cli/ci_logged_errors_detect.py

```diff
@@ -260,6 +264,28 @@ def get_error_logs(log_files: list[str]) -> list[ErrorLog]:
                 ):
                     continue
                 error_logs.append(ErrorLog(match.group(0), log_file))
+            open_panics = {}
+            for line in iter(data.readline, b""):
+                line = line.rstrip(b"\n")
+                if match := PANIC_START_RE.match(line):
+                    service = match.group("service")
+                    assert (
+                        service not in open_panics
+                    ), f"Two panics of same service {service} interleaving: {line}"
```

Comment:
I think this happens in practice -- multiple timely workers may panic together. Or multiple storageds may panic together.

Reply:
That is fine as long as Rust is reasonable enough to order the panic logs in the same process correctly. If Rust doesn't do that for panics in the same process, then I'd consider that a bug, because you'd have no way to figure out which panic message belongs to which panic.